repos/fdbt-site/src/utils/sessions.ts

```diff
@@ -234,6 +235,7 @@ export interface SessionAttributeTypes {
     [VIEW_TIME_RESTRICTION]: ErrorInfo[];
     [VIEW_OPERATOR_GROUP]: ErrorInfo[];
     [VIEW_PRODUCT_GROUP]: ErrorInfo[];
+    [VIEW_CAP]: ErrorInfo[];
```

Comment:
```suggestion
    [VIEW_CAP_ERRORS]: ErrorInfo[];
```

Reply:
Yeah that name makes more sense

Reply:
this is the way the attribute is defined for view pages ...